src/main/plugins/org.dita.pdf2/cfg/common/vars/en.xml

```diff
@@ -368,6 +371,14 @@ See the accompanying LICENSE file for applicable license.
     <variable id="Task Example"></variable>
     <variable id="Task Postreq"></variable>
     
+    <!--Labels for troubleshooting sections, now reused from common-->
+    <variable id="Troubleshooting Condition">Condition or error</variable>
+    <variable id="Troubleshooting Cause">Cause</variable>
+    <variable id="Troubleshooting Remedy">Resolution</variable>
+    <variable id="Troubleshooting responsibleParty">Required personnel</variable>
+    <variable id="Troubleshooting Step">Trouble?</variable>
+    <variable id="Troubleshooting Task">Trouble with this task?</variable>
+    
```

Comment:
Right now, having these only appear in the English file will result in "Cannot resolve variable" errors for other languages. I think that at least until our overall string design changes, new variables have to add a stub element for each language. @jelovirt may have additional thoughts though.